packages/promotion/integration-tests/__tests__/services/promotion-module/campaign.spec.ts

```diff
@@ -163,6 +201,41 @@ describe("Promotion Module Service: Campaigns", () => {
         })
       )
     })
+
+    it("should update promotions of a campaign successfully", async () => {
```

Comment:
suggestion: add a test for removing promotions from a campaign using update

Reply:
good catch, this was broken :D